Sources/SwiftFusion/Inference/ArrayBuffer+Vector.swift

```diff
@@ -158,6 +158,7 @@ extension ArrayBuffer: Vector where Element: Vector {
   @differentiable
   private static func minus (_ lhs: ArrayBuffer, _ rhs: ArrayBuffer) -> ArrayBuffer {
     if rhs.isEmpty { return lhs }
+    if lhs.isEmpty { return .init(rhs.lazy.map { $0.zeroValue - $0 }) }
```

Comment:
Nice catch.

Ugh, we have so many `zeroXXX`s flying around, I can't keep track.  We should think about how to simplify that.

Reply:
Yes :)